akka-actor/src/main/scala/akka/routing/DefaultMetricsBasedResizer.scala

```diff
@@ -0,0 +1,222 @@
+/**
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.routing
+
+import java.time.{ Duration, LocalDateTime }
```

Comment:
we use `scala.concurrent.duration.Duration` / `FiniteDuration` for all other durations, so I think we should stick to that to make APIs consistent